website/contributing/release-branch-cut-and-rc0.md

```diff
@@ -39,6 +39,9 @@ Documents in this section go over steps to run different types of React Native r
 
 - Add and commit the extra file that got created at `sdks/hermes/.hermesversion`.
 
+- Update packages in the monorepo by running `yarn bump-all-updated-packages`. All the package bumps should be a patch. We need to publish the latest package changes, so they will be included in a new RC. Read more about the script and how they work [here](./release-updating-packages).
+- Push the commit created by the previous command to the `0.x-stable` branch.
+
 ### 2. Push the branch and test the current changes
```

Comment:
I just noticed that in this guide we only push the branch up at this step 2.

Maybe let's change this second step to be part of step 1, and separate the Hermes bump into its own step and the bump all packages into its own step too?

Reply:
I created separate steps for Hermes and monorepo packages bumps.

I kept this step as a separate one that we should still run only after Hermes + packages bumps as I don't think it's necessary to push multiple times? Or do you see the workflow as:
- Create new branch, push its current state
- Hermes bump
- monorepo bumps
- push the changes again

Reply:
I think it's ok